html/semantics/interfaces.html

```diff
@@ -27,7 +33,9 @@
 }
 
 elements.forEach(function(a) {
-  do_test(a[0], a[1]);
-  do_test(a[0].toUpperCase(), a[1]);
+  do_test(a[0], a[1], true);
```

Comment:
I'll fix (1)

(2) is because NS preserves case. I suppose I could use NS for both but that's not necessary. Would you prefer a comment justifying NS?